packages/cascara/src/ui/List/List.js

```diff
@@ -9,10 +9,17 @@ const propTypes = {
   data: pt.arrayOf(pt.shape(listItemPropTypes)),
 };
 
+const componentName = 'List';
 const List = ({ data, ...rest }) => {
   const compositeState = useCompositeState();
   return (
-    <Composite {...rest} className={styles._} {...compositeState} as='ul'>
+    <Composite
+      {...rest}
+      className={styles._}
+      {...compositeState}
```

Comment:
```suggestion
      {...compositeState}
      className={styles._}
```
Let's move this spread value above className